setup/_update_all_packages.rst.inc

```diff
@@ -17,3 +17,16 @@ this safely by running:
     breaking changes.
 
 .. _`version constraints`: https://getcomposer.org/doc/articles/versions.md
+
+If you are in a default Symfony application that uses :ref:`Symfony Flex <symfony-flex>`
+and Symfony Recipes, you may also want to reinstall them to their latest version:
+
+.. code-block:: terminal
+
+    $ composer sync-recipes --force
```

Comment:
We should recommend this command `composer recipes:install vendor/package --force -v` to update one by one and `-v` for more information about the update

Reply:
@maxhelias yes i see your idea
but as the upper code is a full `composer update`, that is why i added here a full "recipes update"

the -v option can be added indeed :)

Reply:
So just `composer recipes:install --force -v` will have the same behavior